internal/controllers/clusterextension_controller_test.go

```diff
@@ -332,6 +333,61 @@ func TestClusterExtensionResolutionAndUnpackSuccessfulApplierFails(t *testing.T)
 	require.NoError(t, cl.DeleteAllOf(ctx, &ocv1.ClusterExtension{}))
 }
 
+func TestClusterExtensionServiceAccountNotFound(t *testing.T) {
+	cl, reconciler := newClientAndReconciler(t)
+	reconciler.InstalledBundleGetter = &MockInstalledBundleGetter{
+		err: &authentication.ServiceAccountNotFoundError{
+			ServiceAccountName:      "missing-sa",
+			ServiceAccountNamespace: "default",
+			Err:                     errors.New("error from k8s api"),
```

Comment:
```suggestion
```

Because we change above to remove the error that came back from the k8s API we also need to change here